staging/src/k8s.io/apiextensions-apiserver/go.mod

```diff
@@ -125,6 +125,8 @@ require (
 )
 
 replace (
+	github.com/envoyproxy/go-control-plane => github.com/envoyproxy/go-control-plane v0.9.10-0.20210907150352-cf90f659a021
+	github.com/google/go-cmp => github.com/google/go-cmp v0.5.5
```

Comment:
why these replace?
are those added by the scripts?

Reply:
yeah I just ran the script